### PR TITLE
add rules, quiz and result screens

### PR DESCRIPTION
**Pull request by iRohitGaur**
stv-rules
stv-quiz
stv-result

### CODE REVIEW

File: quiz/quiz.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    />
+    <title>StarTrivia - Quiz - Rohit Gaur</title>
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="/utils/elements.css" />
+    <link rel="stylesheet" href="quiz.css" />
+  </head>
+  <body>
+    <header>
+      <div id="stv-logo" class="stv_logo_wrapper flex_row">
+        <img class="stv_logo" src="/assets/starling.svg" alt="" />
```
Comment (zoef-yt) [inline quiz/quiz.html]: Don't keep empty alt tag

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks amazing. Good to merge.

File: quiz/quiz.html (hunk 1 above)
Comment (iRohitGaur) [reply quiz/quiz.html]: Thanks! Fixed it.

File: quiz/quiz.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            as 11 feet across.
+          </h3>
+          <div
+            class="question_options_wrapper flex_column flex_align_center flex_gap1"
+          >
+            <h4 class="question_option correct_ans">Albatross</h4>
+            <h4 class="question_option">Dove</h4>
+            <h4 class="question_option">Razorbill</h4>
+            <h4 class="question_option">Puffin</h4>
+          </div>
```
Comment (bhtibrewal) [inline quiz/quiz.html]: Suggestion: I don't think Questions/ options should use h tags, they could have been p tags and u could have styled them according to your need.

File: quiz/quiz.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    opt.classList.contains("correct_ans")
+      ? opt.classList.add("right_ans")
+      : opt.classList.add("wrong_ans");
+
+    correctOption.className = "question_option correct_ans right_ans";
+    setTimeout(() => (window.location = "/result/result.html"), 1000);
+  })
+);
+
+const backBtn = document.querySelector("#go-back");
```
Comment (bhtibrewal) [inline quiz/quiz.js]: Suggestion: name the quit button as quitBtn, was confusing for me personally

**Review by rutu14 — APPROVED** [review]
The UI looks clean and stunning.

File: result/result.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+    <div class="landing_wrapper flex_row flex_gap2">
+      <div class="title_wrapper flex_column flex_align_center flex_gap2">
+        <h2 class="app_title">
+          <i class="fas fa-rocket"></i>
+          Final Score: 0/30
+        </h2>
+
+        <div class="quiz_question_wrapper">
+          <h3 class="quiz_question">
```
Comment (bhtibrewal) [inline result/result.html:76]: Suggestion: give question no before each ques in result page also

**Review by bhtibrewal — APPROVED** [review]
Written some minor suggestions, you can look into them. Everything else looks good to merge

File: quiz/quiz.html (hunk 2 above)
Comment (iRohitGaur) [reply quiz/quiz.html]: Thanks for the feedback. Updated!

File: quiz/quiz.js (hunk 3 above)
Comment (iRohitGaur) [reply quiz/quiz.js]: Done!

**Pull request merged**